packages/cryptography/src/Encryption/encryption.config.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+use Tempest\Cryptography\Encryption\EncryptionAlgorithm;
+use Tempest\Cryptography\Encryption\EncryptionConfig;
+
+return new EncryptionConfig(
+    algorithm: EncryptionAlgorithm::AES_256_GCM,
+    key: Tempest\env('SIGNING_KEY', default: ''),
```

Comment:
Should the default be `''`? Making it `null` seems more explicit to me. `null` meaning that the value isn't set and that we cannot encrypt.

Reply:
Indeed! I think I felt that `EncryptionConfig`'s `key` should not be `null`, but an empty string is far worse. I fixed that, thanks!